src/extension/intellisenseHelper.ts

```diff
@@ -126,4 +188,83 @@ export class IntellisenseHelper {
 
         return Q(isRestartRequired);
     }
+
+    /**
+     * Simple check to see if the TypeScript package is in the expected location (where we installed it)
+     */
+    private static isTypeScriptInstalled() : Q.Promise<boolean> {
+        let fileSystem: FileSystem = new FileSystem();
+        let installPath: string = path.join(IntellisenseHelper.getTypeScriptInstallPath(), "lib");
+        return Q.resolve(fileSystem.exists(installPath));
+    }
+
+    /**
+     * Checks for the existance of our installed TypeScript package, if it exists also determine its version
+     */
+    private static getInstalledTypeScriptVersion() : Q.Promise<InstallProps> {
```

Comment:
If I recall correctly, TypeScript doesn't honor access control modifiers on static methods. We should probably just remove them.